.circleci/config.yml

```diff
@@ -19,12 +19,15 @@ setup_env: &setup_env
       name: Setup environment
       command: |
         for i in {1..3}; do
+          echo 'export PATH=/usr/local/cuda/bin:$PATH' >> $BASH_ENV &&
+          source "$BASH_ENV"
           python3.8 --version &&
           python3.8 -m pip install --upgrade pip &&
           cd /home/circleci/project/python &&
           python3.8 setup.py bdist_wheel &&
           sudo python3.8 -m pip install --no-input dist/*.whl &&
           cd /home/circleci/project &&
+          python3.8 -m pip install pycuda &&
```

Comment:
caveat: we need python38-devel (centos) or equivalent package to build pycuda. I'm fine with that, looks like docker images have this package installed by default since CircleCI succeeded. Something to keep in mind if building locally

Reply:
circleCI image `ubuntu-2004-cuda-11.4:202110-01`:
- has `python3-dev` installed
- `/usr/local/cuda/bin` is added to  `PATH` in `config.yml` before we install `pycuda`.

AIT `Dockerfile.cuda`:
- has `RUN apt install -y python3 python3-dev python3-pip` 
- has `ENV PATH=/usr/local/cuda/bin:${PATH}`